.github/workflows/push-to-ghcr.yml

```diff
@@ -0,0 +1,45 @@
+name: Push package to GHCR
+
+on:
+  push:
```

Comment:
Should we limit this to push on main?

Reply:
I had this setup just to see it work.. Now when we know it works it should be enough with the main branch right?